### ISSUE TITLE
Specify both prefix and suffix wildcard for ForbiddenImport pattern

### ISSUE THREAD

**Issue by egor-n** [arturbosch/detekt#1001]
## Context
Is there a way to specify both prefix and suffix wildcard for ForbiddenImport pattern? We want to forbid importing `R.string`, `R.dimen`, etc classes.

## Expected Behavior
The following test should pass:
```kotlin
val code = """
    import com.example.R.string
    import net.example.R.dimen
"""

it("should report both com.example.R.string and net.example.R.dimen") {
	val findings = ForbiddenImport(TestConfig(mapOf(ForbiddenImport.IMPORTS to "*.R.*"))).lint(code)
	assertThat(findings).hasSize(2)
}
```

## Current Behavior
The prefix wildcard is not treated as a wildcard.

## Notes

This change seems to make the test pass:

```diff
class SplitPattern(text: String,
				   delimiters: Regex = regex) {

	private val excludes = text
			.split(delimiters)
			.map { it.trim() }
			.filter { it.isNotBlank() }
+			.map { it.removePrefix("*") }
			.map { it.removeSuffix("*") }
```

**Comment by schalkms** [comment]
Interesting use case. Would you mind submitting a PR with your fix? Thank you!

**Comment by vanniktech** [comment]
Off-topic - wrote a lint rule for exactly that too - https://github.com/vanniktech/lint-rules/blob/master/lint-rules-android-lint/src/main/java/com/vanniktech/lintrules/android/InvalidImportDetector.kt
